### ISSUE TITLE
[Sponsor Page] Sponsor page style adjustment

### ISSUE THREAD

**Issue by anan0202** [pycontw/pycontw-frontend#206]
## Description
Staging版本示意圖：
![image](https://user-images.githubusercontent.com/86242317/160289540-6b55373b-281d-4744-983b-3d5d41e9f6cf.png)

1.中文文案中的您，取消上下引號
2.內文字體大小為 20
3.左右兩邊 margin 為 300
![image](https://user-images.githubusercontent.com/86242317/160289442-9d5b77d2-0b6d-4938-853d-66e9352b3290.png)
4. Margin 左右兩側有背景素材
![image](https://user-images.githubusercontent.com/86242317/160290433-cc11e207-cc52-45eb-ae6a-416dddda7189.png)
5.中文文案須加hyperlink：若想了解 PyCon APAC 2022 完整的贊助方案，歡迎點擊下方按鈕填寫表單留下您的聯絡資料索取贊助書或是直接透過贊助組信箱 sponsorship@python.tw 聯絡我們，讓我們了解貴單位的需求！！
背景素材區：https://drive.google.com/drive/u/1/folders/1mc6Md9X7QRNxYJRKVtcyyocDFOk2n3NQ

**Comment by mattwang44** [comment]
- Issue 1, 2, & 5 will be resolved in #205 
- Issue 4: A standalone issue is created for it (#207)

**Comment by allen91wu** [comment]
英文文案請協助修改為：
```
PyCon Taiwan is driven by our members and you, our awesome sponsor partners! 
Your generous support is the motivation to help us move forward. 
PyCon APAC 2022 calls for your sponsorship to enlarge our community. 

Meanwhile, we also provide the benefit for our sponsors below:
```
中文文案請協助修改為：
```
您的支持是推進 PyCon Taiwan 的強大動力。

PyCon Taiwan 在此召集您的加入，透過贊助以具體行動支持 python 社群的成長！

透過參與 PyCon APAC 2022，我們也提供贊助廠商以下的服務：
```

**Comment by mattwang44** [comment]
closed by #205. The rest issues will be handled in #210 .
